web/src/app/rotations/util.js

```diff
@@ -112,3 +112,35 @@ export function reorderList(_items, oldIndex, newIndex) {
   items.splice(newIndex, 0, _items[oldIndex]) // add dest to newIndex position
   return items
 }
+
+export function getNextHandoffs(count, handoffTime, rotationType, shiftLength) {
```

Comment:
This will need the correct configured zone to return the correct values.

Reply:
I want to say we have a function that handles that discrepancy already (only show the day if there's a diff)

Reply:
That way it's only verbose on those edge cases, and at least it would always be clear & consistent